internal/server/iigointernal/judiciary.go

```diff
@@ -0,0 +1,191 @@
+package iigointernal
+
+import (
+	"github.com/SOMAS2020/SOMAS2020/internal/common/baseclient"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/gamestate"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/roles"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/voting"
+	"github.com/pkg/errors"
+)
+
+type judiciary struct {
+	JudgeID           shared.ClientID
+	budget            shared.Resources
+	presidentSalary   shared.Resources
+	BallotID          int
+	ResAllocID        int
+	speakerID         shared.ClientID
+	presidentID       shared.ClientID
+	EvaluationResults map[shared.ClientID]roles.EvaluationReturn
+	clientJudge       roles.Judge
+}
+
+func (j *judiciary) init() {
```

Comment:
delete? not needed. initial struct without any initialisations will initialise it with default vals.